### DISCUSSION TITLE
Omit stage prefix when using the REST protocol

### DISCUSSION THREAD

**Discussion by jaulz** [brefphp/bref#1653] [Support]
Unfortunately, I still need to use the REST protocol of the API Gateway but that means that there is an ugly stage prefix in the URL. I found out that I can set the Origin path "/${stage}" in the default origin of the CloudFront distribution but I wonder if there is no built-in mechanism?

**Comment by mnapoli** [comment]
I'm not sure what you are asking, you seem to be asking about CloudFront?

The REST API URL (without CloudFront) must contain the stage. It's not something you can remove unfortunately.

**Comment by jaulz** [comment]
Sorry for the duplicate but I think the better place to continue the discussion (of the bug?) in the Lift repository. For anyone interested: https://github.com/getlift/lift/issues/364
